worker/worker.go

```diff
@@ -331,6 +333,16 @@ func (w *Worker) ScheduleJobCompletedJob(jobID string, at int64) (string, error)
 // Start starts the worker
 func (w *Worker) Start() {
 	jobsStatsPort := w.Config.GetInt("workers.statsPort")
-	go workers.StatsServer(jobsStatsPort)
-	workers.Run()
+	http.HandleFunc("/stats", func(w http.ResponseWriter, req *http.Request) {
+		json.NewEncoder(w).Encode(struct {
+			Healthy bool `json:"healthy"`
+		}{
+			Healthy: true,
+		})
+	})
+	if err := http.ListenAndServe(fmt.Sprint(":", jobsStatsPort), nil); err != nil {
```

Comment:
I think ListenAndServe blocks the routine calling it, you need to launch a new goroutine to serve the http server

Reply:
You are correct. Fixing my error.